src/main/java/dk/sdu/se_f22/productmodule/management/ProductJSONReader.java

```diff
@@ -227,7 +226,6 @@ private int findLastOccurence(String whatToFind, String line){
             index--;
         }
```

Comment:
Kunne man ikke bruge "index" istedet for "i" her, da den dekrementeres sammen med i